app-containers/podman/podman-4.6.2.ebuild

```diff
@@ -0,0 +1,135 @@
+	KEYWORDS="~amd64 ~arm64"
+fi
+LICENSE="Apache-2.0 BSD BSD-2 CC-BY-SA-4.0 ISC MIT MPL-2.0"
+SLOT="0"
+IUSE="apparmor btrfs cgroup-hybrid docker +fuse +init +rootless +seccomp selinux systemd"
+RESTRICT="test"
+
+RDEPEND="
+	app-crypt/gpgme:=
+	>=app-containers/conmon-2.0.0
+	>=app-containers/containers-common-0.56.0
+	dev-libs/libassuan:=
+	dev-libs/libgpg-error:=
+	sys-apps/shadow:=
+
+	apparmor? ( sys-libs/libapparmor )
+	btrfs? ( sys-fs/btrfs-progs )
+	cgroup-hybrid? ( >=app-containers/runc-1.0.0_rc6  )
+	!cgroup-hybrid? ( app-containers/crun )
+	docker? ( !app-containers/docker-cli )
+	fuse? ( sys-fs/fuse-overlayfs )
+	init? ( app-containers/catatonit )
+	rootless? ( app-containers/slirp4netns )
+	seccomp? ( sys-libs/libseccomp:= )
+	selinux? ( sec-policy/selinux-podman sys-libs/libselinux:= )
+	systemd? ( sys-apps/systemd:= )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-go/go-md2man
+"
+
+CONFIG_CHECK="
+	~USER_NS
+"
+
+pkg_setup() {
+	use btrfs && CONFIG_CHECK+=" ~BTRFS_FS"
+	linux-info_pkg_setup
+}
+
+src_prepare() {
+	default
+	for file in apparmor_tag btrfs_installed_tag btrfs_tag selinux_tag systemd_tag; do
+		[[ -f hack/"${file}".sh ]] || die
+	done
+
+	for feature in apparmor selinux systemd; do
+		if use "${feature}"; then
+			echo -e "#!/usr/bin/env bash\n echo ${feature}" > hack/"${feature}"_tag.sh || die
+		else
+			echo -e "#!/usr/bin/env bash\n true" > hack/"${feature}"_tag.sh || die
+		fi
+	done
+
+	if use btrfs; then
+		echo -e "#!/usr/bin/env bash\n true" | tee hack/btrfs_installed_tag.sh hack/btrfs_tag.sh || die
+	else
+		echo -e "#!/usr/bin/env bash\n echo exclude_graphdriver_btrfs" > hack/btrfs_installed_tag.sh || die
+		echo -e "#!/usr/bin/env bash\n echo btrfs_noversion" > hack/btrfs_tag.sh || die
+	fi
+
+	use seccomp || eapply "${FILESDIR}"/disable-seccomp-4.6.2.patch
+}
+
+src_compile() {
+	# Filter unsupported linker flags
+	# TODO: Is this really needed? If yes then please uncomment.
+	# filter-flags '-Wl,*'
+
+	# TODO I dont understand why this is needed. Uncomment if something goes wrong
+	# export -n GOCACHE GOPATH XDG_CACHE_HOME
+
+	export PREFIX="${EPREFIX}/usr"
+	emake BUILDFLAGS="-v -work -x" GOMD2MAN="go-md2man"
+	use docker && emake BUILDFLAGS="-v -work -x" GOMD2MAN="go-md2man" docker-docs
+
+}
+
+src_install() {
+	default
+	emake DESTDIR="${D}" install.completions
+	use docker && emake DESTDIR="${D}" install.docker-full
+
+	if use cgroup-hybrid; then
+		insinto /etc/cni/net.d
+		doins cni/87-podman-bridge.conflist
+	fi
+
+	newconfd "${FILESDIR}"/podman.confd podman
+	newinitd "${FILESDIR}"/podman.initd podman
+
+	insinto /etc/logrotate.d
+	newins "${FILESDIR}/podman.logrotated" podman
+
+	keepdir /var/lib/containers
+}
+
+pkg_preinst() {
+	PODMAN_ROOTLESS_UPGRADE=false
+	if use rootless; then
+		has_version 'app-containers/podman[rootless]' || PODMAN_ROOTLESS_UPGRADE=true
+	fi
+}
+
+pkg_postinst() {
+	tmpfiles_process podman.conf
+	use docker && tmpfiles_process podman-docker.conf
```

Comment:
meh, ok, could do usev above in the single tmpfiles_process call but it doesn't matter much here